src/builder.rs

```diff
@@ -183,6 +183,36 @@ impl SourceMapBuilder {
         raw
     }
 
+    /// Adds a new mapping to the builder.
+    pub fn add_raw(
+        &mut self,
+        dst_line: u32,
+        dst_col: u32,
+        src_line: u32,
+        src_col: u32,
+        source: Option<u32>,
+        name: Option<u32>,
+    ) -> RawToken {
+        let src_id = match source {
+            Some(source) => source,
```

Comment:
Should this index be validated? The `!0` suggests we don’t really care?

Reply:
It's same as add.

https://github.com/getsentry/rust-sourcemap/blob/97cf8ebc536c7cc53e1f103086e8d0f150470ae6/src/builder.rs#L166-L172